src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/UrlRewriteFileParser.cs

```diff
@@ -207,10 +207,17 @@ private void ParseUrlAction(XElement urlAction, UrlRewriteRuleBuilder builder, b
             }
 
             RedirectType redirectType;
-            if (!Enum.TryParse(urlAction.Attribute(RewriteTags.RedirectType)?.Value, true, out redirectType))
+            if(urlAction.Attribute(RewriteTags.RedirectType) == null)
             {
                 redirectType = RedirectType.Permanent;
             }
+            else
+            {
+                if(!Enum.TryParse(urlAction.Attribute(RewriteTags.RedirectType)?.Value, ignoreCase: true, result: out redirectType))
```

Comment:
Better. Don't all the other enum parsers in this PR have the same issue with defaults?